packages/cli-repl/src/mongosh-repl.ts

```diff
@@ -839,16 +842,16 @@ class MongoshNodeRepl implements EvaluationListener {
         (this.runtimeState().repl as any).historySize = value;
       }
       if (key === 'inspectCompact') {
-        this.inspectCompact = value as CliUserConfig['inspectCompact'];
+        this.inspectCompact = value as number|boolean;
       }
       if (key === 'inspectDepth') {
-        this.inspectDepth = value as CliUserConfig['inspectDepth'];
+        this.inspectDepth = value as number;
       }
       if (key === 'showStackTraces') {
-        this.showStackTraces = value as CliUserConfig['showStackTraces'];
+        this.showStackTraces = value as boolean;
       }
       if (key === 'redactHistory') {
-        this.redactHistory = value as CliUserConfig['redactHistory'];
+        this.redactHistory = value as 'keep' | 'remove' | 'remove-redact';
```

Comment:
These are odd. Should it be inlining the types here?

Reply:
Oh I think I know how that happened. --fix probably removed the "unnecessary" bit and then I added something back to make something work. I can just undo the change now.